data/blog/new-features-in-v1.mdx

```diff
@@ -7,6 +7,7 @@ draft: false
 summary: 'An overview of the new features released in v1 - code block copy, multiple authors, frontmatter layout and more'
 layout: PostSimple
 bibliography: references-data.bib
+canonicalUrl: https://example.com/
```

Comment:
Let's not change my blog post canonical link :)

Reply:
Sure thing!